flake.nix

```diff
@@ -105,6 +105,44 @@
             vendorSha256 = "sha256-f/DIAw8XWb1osfXAJ/ZKsB0sOmFnJincAQlfVHqElBE=";
           };
 
+          # Apache Tika server 
+          tika-server = with final; stdenv.mkDerivation rec {
+            name = "tika-server-1.26";
+  
+            src = fetchurl {
+              url = https://archive.apache.org/dist/tika/tika-server-1.26.jar;
```

Comment:
I would have prefered to have the src as an input. That makes easier updates.

Reply:
I just realised that this was the jar file, so this won't work as an input. I'll leave the fetchurl as is then.

Reply:
If you are interested, I just found the source of the git repo https://github.com/apache/tika.git
I think ideally it should be built from source. I understand though that building java might be a pain, so no worries if you don't have time.